backend/seed.js

```diff
@@ -167,7 +168,7 @@ const seed = async () => {
             fakerFR.word.adjective(),
             faker.helpers.arrayElement(skillLevel, 1),
             fakerFR.number.int({ min: 1, max: 20 }),
-            fakerFR.number.int({ min: 1, max: 20 }),
+            fakerFR.number.int({ min: 1, max: 200 }),
```

Comment:
1557 job created in loop, set max 1557

Reply:
we don't need to have 1557 sector, a sector can take many job offers